.github/workflows/package.yml

```diff
@@ -42,16 +31,26 @@ jobs:
           - "3.10"
           - "3.11"
           - "3.12"
+          - "3.13"
           - "pypy-3.7"
+          - "pypy-3.9"
+          - "pypy-3.10"
     steps:
     - uses: actions/checkout@v4
+    - name: Install the latest version of uv
+      uses: astral-sh/setup-uv@v4
```

Comment:
On occasions where I've used this action, I've removed/replaced the `actions/setup-python` step and used `uv python install` instead, which gets an [indygreg standalone](https://github.com/astral-sh/python-build-standalone). However, this may not be an option for Python 3.7, so perhaps keeping setup-python is best.